src/scss/settings/vars.scss

```diff
@@ -1 +1,42 @@
-// colors
\ No newline at end of file
+// colors
+
+//primary-colors
+$white:#ffff;
+$green:#16E024;
+$black:#000000;
+
+
+//secondary colors
+
+//glicth
+$white-blue:#00dffd;
+$aqua:#2af598;
+$pink:#fd39d2;
+
+
+//hover btn artist
+
+$blue:#2616e0;
+$turquoise:#00DFFD;
+
+// typography
+$regular-txt: 'Lexend Deca', sans-serif;
+//The wrong
```

Comment:
$the-wrong donde se va a usar? si se usa solo para el logo, se puede limitar la cantidad de caracteres que se cargan usando el parametro text

Reply:
En el footer, es la tipografía oficial de ellos, es mejor que la quite?

Reply:
solo se usa para eso? y solo se usa en 1 lugar?